content/3_Silver/Sorting_Custom.mdx

```diff
@@ -653,6 +654,105 @@ public class Sol {
 
 </LanguageSection>
 
```

Comment:
what about the coordinate compression where spacing should be maintained asw well? tho i'm not sure if those showed up in silver yet.